### ISSUE TITLE
New Design Needed for Senate

### ISSUE THREAD

**Issue by thanisara-r** [wevisdemo/parliament-watch#144]
**Is your feature request related to a problem? Please describe.**
Senate's info is grouped by the former approach and cannot be applied to the existing senators.
<img width="1422" alt="image" src="https://github.com/user-attachments/assets/ddc14bf1-88db-47b1-8006-728031544968">

<img width="1424" alt="image" src="https://github.com/user-attachments/assets/d6cd7d0d-4e09-4284-a276-09e94180ce4d">

**Describe the solution you'd like**
Group the current senate by provinces?

**Comment by beambeambeam** [comment]
Suggest based on 20 social and professional groups, as it's a role that they pick to be in the Senate.
And provinces should be subordinate to the Senate election district? but still, an idea of provinces will be much easier to understand.

**Comment by Th1nkK1D** [comment]
> Suggest based on 20 social and professional groups, as it's a role that they pick to be in the Senate. And provinces should be subordinate to the Senate election district? but still, an idea of provinces will be much easier to understand.

I agree with 20 social and professional groups and provinces that are used when senate candidates register for the election.

From the data perspective, we do have `appointmentMethod` and `province` columns. The easiest fix would be to put professional groups in `appointmentMethod` and their registered province as `province`. For the long term, we might need to think if the term `appointmentMethod` is generalized enough.
![image](https://github.com/user-attachments/assets/5bb95609-853f-4490-86e8-df7f02ecbe67)

**Comment by Th1nkK1D** [comment]
> ** Senate's info is grouped by the former approach and cannot be applied to the existing senators. 

@thanisara-r It shows no data because we literally have no data in the `appointmentMethod` and `province` columns for the latest term of Senate. (We should try adding data first, then there might be some code we need to change)

![image](https://github.com/user-attachments/assets/d0e39dac-34bb-444c-a5a6-78141fcd8dd5)

**Comment by Th1nkK1D** [comment]
The data has been update and now we are supporting dynamic color for unpredictable senate groups in the future in 50dc7882ff2f7fb94ac0abcb650aebd187ea9ba2
